ci/travis_script.sh

```diff
@@ -60,5 +64,13 @@ if [ "${PYSMT_SOLVER}" == "all" ];
 then
     python install.py --msat --conf --force;
     cp -v $(find ~/.smt_solvers/ -name mathsat -type f) /tmp/mathsat;
-    (for ex in `ls examples/*.py`; do echo $ex; python $ex || exit $?; done);
+
+    # since we're relying on relative `pysmt` import in examples,
+    # ensure `.` is added to `sys.path`
+    export PYTHONPATH=${PYTHONPATH:-.}
```

Comment:
Why wasn't this a problem before?

Would it make sense to pip install the library locally?

Reply:
This wasn't a problem before because before `PYTHONPATH` was always explicitly set.

Absolutely, local `pip install .` is an option. I went with this solution because it required minimal changes (to stay as close to the original solution).

Reply:
``pip install`` might be the cleaner version, but let's keep that for another time.